packages/sql-graphql/lib/relationship.js

```diff
@@ -51,7 +51,8 @@ module.exports = function establishRelations (app, relations, resolvers, loaders
         resolvers[foreign.type] = resolvers[foreign.type] || {}
         resolvers[foreign.type][lowered] = async function (obj, args, ctx, info) {
           const fields = fromSelectionSet(info.fieldNodes[0].selectionSet, new Set())
-          const toSearch = { ...args, fields: [...fields], ctx }
+          const relationalFields = current.entity.relations.map((relation) => relation.column_name)
```

Comment:
Should I perhaps move this out of the relation loop to avoid looping on every iteration?

Reply:
I've updated to only include the current relation in the loop 👍